resourcemanager/util/util.go

```diff
@@ -18,20 +18,10 @@ import "time"
 
 // GorotinuePool is a pool interface
 type GorotinuePool interface {
-	Release()
+	ReleaseAndWait()
 	Tune(size int)
 	LastTunerTs() time.Time
-	MaxInFlight() int64
-	InFlight() int64
-	MinRT() uint64
-	MaxPASS() uint64
 	Cap() int
-	// LongRTT is to represent the baseline latency by tracking a measurement of the long term, less volatile RTT.
-	LongRTT() float64
-	UpdateLongRTT(f func(float64) float64)
-	// ShortRTT is to represent the current system latency by tracking a measurement of the short time, and more volatile RTT.
-	ShortRTT() uint64
-	GetQueueSize() int64
```

Comment:
Some API will be used for statistics. But it will be released in this sprint. so it is removed.